api/src/services/script-runner.service.ts

```diff
@@ -1910,4 +1910,572 @@ export class ScriptRunnerService {
       active: false,
     },
   ];
+
+  /**
+   *
+   * @param req incoming request object
+   * @returns successDTO
+   * @description resets all non-english translations
+   */
+  async resetMissingTranslations(req: ExpressRequest): Promise<SuccessDTO> {
+    const requestingUser = mapTo(User, req['user']);
+    await this.markScriptAsRunStart(
+      'reset missing translations',
+      requestingUser,
+    );
+
+    await this.prisma.translations.update({
+      where: {
+        jurisdictionId_language: {
+          jurisdictionId: null,
+          language: 'es',
+        },
```

Comment:
issue: I'm getting a 500 when running the script. Looks like it's because the `jurisdictionId_language` combo requires a jurisdictionId and it can't be null. You could do a look up first to get the id, or the better route would be to use the `updateTranslationsForLanguage` function that is in this file.